### ISSUE TITLE
Aria-sort doesn't work in edge

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#6978]
Visit https://dequeuniversity.com/library/aria/tables/sf-sortable-grid
Move to the table.
Now, read the column headers, and click one.
In Narrator, aria-sort is properly indicated (But not in scan mode). In NVDA, this information is omitted.

**Comment by derekriemer** [comment]
CC @michaelDCurran

**Comment by jcsteh** [comment]
aria-sort is only exposed in UIA via the AriaProperties property.

**Comment by Adriani90** [comment]
This works as expected with NVDA Version: alpha-20015,56a62097, Firefox 75, Crome 81and Edge Cromium 81. Closing as works for me.

**Comment by Adriani90** [comment]
Adding appropriate label since this is probably still reproducible in Edge Spartan and UIA related environments.
